### PR TITLE
Move everything in 2.5.0 from 2.x to 2.5 branch in manifest

### LINKED ISSUE #2908: [RELEASE] Release version 2.5.0
## Release OpenSearch and OpenSearch Dashboards 2.5.0

I noticed that a manifest was automatically created in [manifests/2.5.0](/opensearch-project/opensearch-build/tree/main/manifests/2.5.0). Please follow the following checklist to make a release.

<details><summary>How to use this issue</summary>
<p>

## This Release Issue

This issue captures the state of the OpenSearch release, its assignee is responsible for driving the release. Please contact them or @mention them on this issue for help. There are linked issues on components of the release where individual components can be tracked.  More details are included in the Maintainers [Release owner](https://github.com/opensearch-project/opensearch-build/blob/main/MAINTAINERS.md#release-owner) section.

## Release Steps

There are several steps to the release process, these steps are completed as the whole release and components that are behind present risk to the release.  The release owner completes the tasks in this ticket, whereas component owners resolve tasks on their ticket in their repositories.

Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.

### Component List

To aid in understanding the state of the release there is a table with status indicating each component state. This is updated based on the status of the component issues.

</p>
</details>

### Preparation

- [x] Assign this issue to a release owner.
- [x] Declare a pencils down date for new features to be merged. 
- [x] **2023/01/10** is pencils down date for code freeze.
- [x] Update the Campaigns section to include monitoring campaigns during this release. 
- [x] Update this issue so all `__REPLACE_RELEASE-__` placeholders have actual dates.
- [x] Document any new quality requirements or changes.
- [x] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1). 
- [x] [Create a release issue in every component repo](https://github.com/opensearch-project/opensearch-build/blob/main/meta/README.md#create-a-release-issue) that links back to this issue, update Components section with these links.
- [x] Ensure the label is created in each component repo for this new version, and the next minor release. [Create a version label](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-or-update-labels-in-all-plugin-repos)
- [x] Ensure that all release issues created above are assigned to an owner in the component team.
- [x] Increase the build frequency for the this release from once a day (H 1 * * *) to once every hour (H/60 * * * *) in [jenkinsFile](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/check-for-build.jenkinsfile)

### Campaigns
- [ ] https://github.com/opensearch-project/opensearch-plugins/issues/95
- [ ] https://github.com/opensearch-project/opensearch-plugins/issues/138
- [ ] https://github.com/opensearch-project/opensearch-plugins/issues/139
- [ ] https://github.com/opensearch-project/opensearch-build/issues/58
- [ ] https://github.com/opensearch-project/opensearch-plugins/issues/142

### Release Readiness

GitHub issue custom search

<html><head></head><body>

Repo | Untriaged issues | Issues tagged to 2.5 version
-- | -- | --
OpenSearch |  [](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
OpenSearch-Dashboards |  [](https://github.com/opensearch-project/OpenSearch-Dashboards/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/OpenSearch-Dashboards/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/OpenSearch-Dashboards/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
alerting |  [](https://github.com/opensearch-project/alerting/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/alerting/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/alerting/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
anomaly-detection |  [](https://github.com/opensearch-project/anomaly-detection/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/anomaly-detection/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/anomaly-detection/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
asynchronous-search |  [](https://github.com/opensearch-project/asynchronous-search/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/asynchronous-search/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/asynchronous-search/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
common-utils |  [](https://github.com/opensearch-project/common-utils/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/common-utils/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/common-utils/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
cross-cluster-replication |  [](https://github.com/opensearch-project/cross-cluster-replication/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/cross-cluster-replication/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/cross-cluster-replication/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
reporting |  [](https://github.com/opensearch-project/reporting/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/reporting/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/reporting/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
dashboards-reporting |  [](https://github.com/opensearch-project/dashboards-reporting/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/dashboards-reporting/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/dashboards-reporting/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
geospatial |  [](https://github.com/opensearch-project/geospatial/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/geospatial/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/geospatial/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
index-management |  [](https://github.com/opensearch-project/index-management/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/index-management/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/index-management/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
job-scheduler |  [](https://github.com/opensearch-project/job-scheduler/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/job-scheduler/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/job-scheduler/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
k-NN |  [](https://github.com/opensearch-project/k-NN/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/k-NN/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/k-NN/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
ml-commons |  [](https://github.com/opensearch-project/ml-commons/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/ml-commons/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/ml-commons/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
notifications |  [](https://github.com/opensearch-project/notifications/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/notifications/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/notifications/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
dashboards-notifications |  [](https://github.com/opensearch-project/dashboards-notifications/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/dashboards-notifications/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/dashboards-notifications/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
observability |  [](https://github.com/opensearch-project/observability/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/observability/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/observability/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
dashboards-observability |  [](https://github.com/opensearch-project/dashboards-observability/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/dashboards-observability/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/dashboards-observability/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
performance-analyzer |  [](https://github.com/opensearch-project/performance-analyzer/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/performance-analyzer/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/performance-analyzer/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
performance-analyzer-rca |  [](https://github.com/opensearch-project/performance-analyzer-rca/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/performance-analyzer-rca/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/performance-analyzer-rca/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
security |  [](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
sql |  [](https://github.com/opensearch-project/sql/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/sql/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/sql/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
dashboards-query-workbench |  [](https://github.com/opensearch-project/dashboards-query-workbench/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/dashboards-query-workbench/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/dashboards-query-workbench/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
alerting-dashboards-plugin |  [](https://github.com/opensearch-project/alerting-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/alerting-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/alerting-dashboards-plugin/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
anomaly-detection-dashboards-plugin |  [](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
dashboards-maps |  [](https://github.com/opensearch-project/dashboards-maps/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/dashboards-maps/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/dashboards-maps/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
index-management-dashboards-plugin |  [](https://github.com/opensearch-project/index-management-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/index-management-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/index-management-dashboards-plugin/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
dashboards-visualizations |  [](https://github.com/opensearch-project/dashboards-visualizations/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/dashboards-visualizations/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/dashboards-visualizations/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
opensearch-dashboards-functional-test |  [](https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
security-dashboards-plugin |  [](https://github.com/opensearch-project/security-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/security-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/security-dashboards-plugin/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
security-analytics |  [](https://github.com/opensearch-project/security-analytics/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/security-analytics/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/security-analytics/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
security-analytics-dashboards-plugin |  [](https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
neural-search |  [](https://github.com/opensearch-project/neural-search/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/neural-search/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/neural-search/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)
dashboards-search-relevance |  [](https://github.com/opensearch-project/dashboards-search-relevance/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) |  [](https://github.com/opensearch-project/dashboards-search-relevance/issues?q=is%3Aissue+is%3Aopen+label%3Av2.5.0) [](https://github.com/opensearch-project/dashboards-search-relevance/issues?q=is%3Aissue+is%3Aclosed+label%3Av2.5.0)

</body></html>

### Release branch readiness - **Ends 2023/01/06**
- [x] Component versions are auto-incremented to `2.5.0` version
- [x] Plugins team to ensure working CI in component repositories and merge the version increment PR's.
- [x] OpenSearch and OpenSearch-Dashboards core create `2.5` release branch
- [x] All components create `2.5` branch for the release.

### CI/CD Freeze - **Ends 2023/01/09**
- [x] OpenSearch Engineering Effectiveness team will update the input manifests to point to `2.5` branch and configure the CI to run against `2.5` branch.
- [x]  Create Jenkins workflows that run daily snapshot builds for OpenSearch and OpenSearch Dashboards. 

### Code Complete - **Ends 2023/01/10**
- [x] Code Complete: Make sure that the code for this specific version of the release is ready and the branch corresponding to this release has been added to this release version manifest.
- [x] OpenSearch / OpenSearch-Dashboards core and components teams finalize their features
- [x] Verify pull requests to add each component to [manifests/2.5.0/opensearch-2.5.0.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.5.0/opensearch-2.5.0.yml) and [manifests/2.5.0/opensearch-dashboards-2.5.0.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.5.0/opensearch-dashboards-2.5.0.yml) have been merged.
- [x] Gather, review and combine the release notes from components repositories.
  * ﻿﻿#3115

### Release testing - **2023/01/11 - 2023/01/13**
- [x] Declare a release candidate build, and provide the instructions with the release candidates for teams on testing (2022/11/11).
- [x] Stop builds for this version of OpenSearch and/or OpenSearch Dashboards in order to avoid accidental commits going in unknowingly. Restart only if necessary else manually run the build workflow and declare new release candidate.
- [x] Sanity Testing (2023/01/12 - 2023/01/13): Sanity testing and fixing of critical issues found by teams. Teams test their components within the distribution, ensuring integration, backwards compatibility, and perf tests pass.
- [x] Publish all test results in the comments of this issue.

### Performance testing validation - **2023/01/12 - 2023/01/13**
- [x] Performance tests do not show a regression
- [x] Longevity tests do not show any issues

### Release - ~~2023/01/16 - 2023/01/17~~ 2023/01/18 - 2023/01/19
- [x] Verify [all issues labeled `v2.5.0` in all projects](https://github.com/opensearch-project/project-meta#find-labeled-issues) have been resolved.
- [ ] Complete [documentation](https://github.com/opensearch-project/documentation-website) for this release.
- [x] Author [blog post](https://github.com/opensearch-project/project-website) for this release.
- [ ] 2023/01/16 - Publish this release on [opensearch.org](https://opensearch.org/downloads.html).
- [ ] 2023/01/17 - Publish a [blog post](https://github.com/opensearch-project/project-website) - release is launched!

### Post Release

- [x] Create [release tags](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/release-tag/release-tag.jenkinsfile) for each component (Jenkins job name: release-tag-creation).
- [x] Replace refs in [manifests/2.5.0](/opensearch-project/opensearch-build/tree/main/manifests/2.5.0) with tags and remove checks.
- [x] If this is a major or minor version release, stop building previous patch version.
- [x] Generate distribution release notes reviewed by PM team for opensearch-build repository.
- [x] Increment version for Helm Charts [(sample PR)](https://github.com/opensearch-project/helm-charts/pull/246) for the `2.5.0` release.
  * https://github.com/opensearch-project/helm-charts/pull/373
- [x] Increment version for Ansible Charts [(sample PR)](https://github.com/opensearch-project/ansible-playbook/pull/50) for the `2.5.0` release.
  * https://github.com/opensearch-project/ansible-playbook/pull/109
- [x] Prepare [for next patch release](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#increment-a-version-in-every-plugin) by incrementing patch versions for each component.
  - https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3313
  - https://github.com/opensearch-project/OpenSearch/pull/5991
- [x] Update [this template](https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md) with any new or missed steps.
- [x] Create an issue for a retrospective, solicit feedback, and publish a summary.
  * https://github.com/opensearch-project/opensearch-build/issues/3117

### Components
| Component | On track | Notes |
| --------- | -------- | ----- |
| https://github.com/opensearch-project/OpenSearch/issues/5705 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3174 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/alerting/issues/732 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/anomaly-detection/issues/769 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/asynchronous-search/issues/217 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/reporting/issues/617 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/dashboards-reporting/issues/19 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/dashboards-visualizations/issues/152 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/index-management/issues/641 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/job-scheduler/issues/293 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/k-NN/issues/708 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/performance-analyzer/issues/358 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/security-dashboards-plugin/issues/1286 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/security/issues/2373 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/sql/issues/1221 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/dashboards-query-workbench/issues/19 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/observability/issues/1365 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/dashboards-observability/issues/178 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/alerting-dashboards-plugin/issues/434 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/notifications/issues/588 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/dashboards-notifications/issues/4 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/index-management-dashboards-plugin/issues/536 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/issues/385 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/cross-cluster-replication/issues/672 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/common-utils/issues/342 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/ml-commons/issues/658 | :green_circle: | In Manifest   |
| https://github.com/opensearch-project/geospatial/issues/199 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/dashboards-maps/issues/160 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/security-analytics/issues/230 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/283 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/neural-search/issues/93 | :green_circle: | In Manifest |
| https://github.com/opensearch-project/dashboards-search-relevance/issues/106 | :green_circle: | In Manifest |

<details><summary>Legend</summary>
<p>

| Symbol | Meaning |
| -------- | ---------- |
| :green_circle: | On track with overall release |
| :yellow_circle: | Missed last milestone |
| :red_circle: | Missed multiple milestones |

</p>
</details>

Issues:
* https://github.com/opensearch-project/opensearch-build/issues/3116

PRs:
* https://github.com/opensearch-project/opensearch-build/pull/3061
* https://github.com/opensearch-project/opensearch-build/pull/3060
* #3090 
* https://github.com/opensearch-project/opensearch-build-libraries/pull/109
* https://github.com/opensearch-project/opensearch-build/pull/3095
* https://github.com/opensearch-project/opensearch-build/pull/3096
* https://github.com/opensearch-project/opensearch-build/pull/3114
* https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/450
  * https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/453

### PR DESCRIPTION
**Pull request by peterzhuamazon**
Signed-off-by: Peter Zhu <zhujiaxi@amazon.com>

### Description
Move everything in 2.5.0 from 2.x to 2.5 branch in manifest

### Issues Resolved
#2908

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/2.5.0/opensearch-dashboards-2.5.0.yml [hunk 1]
```diff
@@ -12,27 +12,27 @@ components:
     ref: '2.5'
   - name: searchRelevanceDashboards
     repository: https://github.com/opensearch-project/dashboards-search-relevance.git
-    ref: 2.x
+    ref: '2.5'
```
Comment (zelinh) [inline manifests/2.5.0/opensearch-dashboards-2.5.0.yml:15]: Seems like this repo doesn't cut 2.5 branch yet.

Comment (peterzhuamazon) [reply manifests/2.5.0/opensearch-dashboards-2.5.0.yml:15]: I already contacted their release owner she is on it now.
@mingshl

Comment (mingshl) [reply manifests/2.5.0/opensearch-dashboards-2.5.0.yml:15]: cut 2.5 branch https://github.com/opensearch-project/dashboards-search-relevance/tree/2.5

Comment (peterzhuamazon) [reply manifests/2.5.0/opensearch-dashboards-2.5.0.yml:15]: Thanks @mingshl !

**Review by gaiksaya — APPROVED** [review]

**Review by rishabh6788 — APPROVED** [review]

**Pull request merged**